backend/Squirrel.Core/Squirrel.Core.WebAPI/Extensions/ServiceCollectionExtensions.cs

```diff
@@ -33,6 +33,14 @@ public static void RegisterCustomServices(this IServiceCollection services)
         services.AddScoped<IUserService, UserService>();
         services.AddScoped<IAuthService, AuthService>();
         services.AddScoped<IProjectService, ProjectService>();
+        services.AddUserIdStorage();
+    }
+
+    public static void AddUserIdStorage(this IServiceCollection services)
+    {
+        services.AddScoped<UserIdStorageService>();
+        services.AddTransient<IUserIdSetter>(s => s.GetRequiredService<UserIdStorageService>());
+        services.AddTransient<IUserIdGetter>(s => s.GetRequiredService<UserIdStorageService>());
```

Comment:
why do we use 3 services for 1 purpose? maybe one would be enough for saving userId?